taxi/views.py

```diff
@@ -22,33 +20,87 @@ def index(request):
         "num_cars": num_cars,
         "num_manufacturers": num_manufacturers,
         "num_visits": num_visits + 1,
+        "url_home": True
     }
 
     return render(request, "taxi/index.html", context=context)
 
 
 class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
+    extra_context = {"url_manufacturer": True}
     context_object_name = "manufacturer_list"
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    extra_context = {"url_manufacturer": True}
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
```

Comment:
Don't override template_name, context_object_name and so on if they are the same as default ones -- checklist